application/common/components/passwordStore/Multiple.php

```diff
@@ -165,4 +159,12 @@ public function assess($employeeId, $password)
         }
         return true;
     }
+
+    /**
+     * {@inheritdoc}
```

Comment:
This is closer to a preference than an objective bit of feedback, but I've found that `{@inheritdoc}` doesn't end up adding any significant value after all. PhpStorm seems to show the overridden function's documentation on Ctrl-Q even without that. However, you may have some clear benefit in mind. Either way is okay with me.

Reply:
Not much thought, other than that I like the visual separation between methods.